v1/protocols/ipfs.ts

```diff
@@ -121,6 +121,8 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
       throw new Error('IPFS must be initialized using load() before calling sync()')
     }
 
+    // By default, inline empty directory CID
+    let toPublish = '/ipfs/bafyaabakaieac/'
```

Comment:
modern day magic numbers haha

Reply:
I think there's literally a handfull of people that understand what the hell this CID means. I'm not certain I am among them. :P